app/views/group/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<% provide(:title, "Categories") %>
+<div class='card'>
+  <div class="border border-light shadow-sm">
+      <p class="text-muted fs-5 py-3 ps-4">All Categories:</p>
+  </div>
+  <% if @groups.empty? %>
+      <p class="text-muted fs-5 py-3 ps-4">Add your first category to start.</p>
+  <% end %>
+  <% @groups.each do |group|%>  
+      <% if group.user_id == @user %>
+        <div class="card-body container">
+          <div class="card justify-content-center shadow rounded-3">
+            <div class="row g-0">
+              <div class="col-3">
+                <img src=<%= group.icon %> class="img-fluid rounded-start p-4" alt="...">
+              </div>

```

Comment:
- [ ] The relative position of the image compared to the parent card element doesn't represent the original design. Carefully compare the positioning and margin. :pray: 

![image](https://user-images.githubusercontent.com/64894927/144660681-021f5063-7dd7-425a-9c74-fc713c700ca0.png)

![image](https://user-images.githubusercontent.com/64894927/144660721-62d5a778-0c41-44e7-824c-b22efd5b4390.png)